From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 2 Apr 2015 11:46:08 -0600
Subject: [PATCH] Add cinder's tgt config

Signed-off-by: Alberto Murillo <alberto.murillo.silva@intel.com>
---
 cinder.tgt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 cinder.tgt

diff --git a/cinder.tgt b/cinder.tgt
new file mode 100644
index 0000000..444eb00
--- /dev/null
+++ b/cinder.tgt
@@ -0,0 +1 @@
+include /var/lib/cinder/volumes/*
